docs/topics/wasm-wasi.md

```diff
@@ -0,0 +1,127 @@
+[//]: # (title: Get started with Kotlin/Wasm and WASI)
+
+> Kotlin/Wasm is [Alpha](components-stability.md). It may be changed at any time.
+>
+> [Join the Kotlin/Wasm community.](https://slack-chats.kotlinlang.org/c/webassembly)
+>
+{style="note"}
+
+This tutorial demonstrates how to run a [WASI](https://wasi.dev/) application with [Kotlin/Wasm](wasm-overview.md) in IntelliJ IDEA. 
+The output is a simple application that utilizes the WASI API in [Node.js](https://nodejs.org/en), [Deno](https://deno.com/), 
```

Comment:
The output/application has nothing to do with Node/Deno/WasmEdge, it utilizes the **standard** WASI API.
We show how we can run this it all these VMs, as an example.